timesketch/frontend-ng/src/views/Explore.vue

```diff
@@ -457,7 +493,11 @@ limitations under the License.
                 @update:options="updateOptions"
                 :show-current-page="true"
                 :items-per-page-options="[10, 40, 80, 100, 200, 500]"
-              ></v-data-footer>
+              >
+                <template v-slot:prepend v-if="selectedEvents.length">
+                  {{ selectedEvents.length }} events selected. Action menu to appear here..
```

Comment:
`Action menu to appear here..` sounds like a debug entry. Is this supposed to be here?

Reply:
This has since been removed.